README.md

```diff
@@ -28,22 +28,79 @@ implement user experiences consistent with
 To get started using Canvas kit React first add or install the module to your existing React project
 
 ```sh
-yarn add @workday/canvas-kit-react
+yarn add @workday/canvas-kit-react @workday/canvas-tokens-web @workday/canvas-kit-react-fonts
 ```
 
 or
 
 ```sh
-npm install @workday/canvas-kit-react
+npm install @workday/canvas-kit-react @workday/canvas-tokens-web @workday/canvas-kit-react-fonts
 ```
```

Comment:
Question: Does everyone _always_ need CKR Fonts? I think you would only _need_ it if Roboto isn't being pulled in for you. Most teams using CKR have fonts already sorted for them. Maybe we should add a caveat. Something like:

If your application environment does not already provide Roboto for you, you can also install `@workday/canvas-kit-react-fonts`.

```sh
npm install @workday/canvas-kit-react-fonts
```

The example below uses `@workday/canvas-kit-react-fonts`, but you can omit those lines if you don't need that package.

Reply:
yeah that's fair, I'll update.

Reply:
updated!